utils/build-script-impl

```diff
@@ -1281,6 +1281,36 @@ function cmake_config_opt() {
     fi
 }
 
+function copy_embedded_compiler_rt_builtins_from_darwin_host_toolchain() {
+    local clang_dest_dir=$1
+
+    HOST_CXX_DIR=$(dirname "${HOST_CXX}")
+    HOST_LIB_CLANG_DIR="${HOST_CXX_DIR}/../lib/clang"
+    DEST_LIB_CLANG_DIR="${clang_dest_dir}/lib/clang"
+
+    if [[ -d "${HOST_LIB_CLANG_DIR}" ]] && [[ -d "${DEST_LIB_CLANG_DIR}" ]]; then
```

Comment:
Would be nicer to read if this was made into an early return on failure.

Reply:
Addressed in 0ca06c2